src/edu/ucsb/cs56/projects/FTP_client/ClientGui.java

```diff
@@ -23,6 +23,7 @@
 	private JPanel loginPanel;
 	private JPanel downloadPanel;
 	private JButton connectButton;
+    private JButton cdButton;
```

Comment:
Be careful about tabs vs spaces!  You've indented with spaces, but the rest of the project uses tabs, which can cause the code to look weird on other computers.